CONTRIBUTING.md

```diff
@@ -9,6 +9,7 @@ If you are new to contributing to open source, please read the Open Source Guide
 <!-- toc -->
 
 - [Code of Conduct](#code-of-conduct)
+- [Videos](#videos)
```

Comment:
Is this change required? Please give some context.

Reply:
was added automatically, I have removed it.